vignettes/Model.Rmd

```diff
@@ -10,90 +12,365 @@ vignette: >
+
+### Parameters
+
+Parameters shown on the infographic include:
+
+-   $\Lambda_i$: the age-specific (*i*) force of infection
+-   $\phi_i$: the age-specific (*i*) probability of clinical disease
+-   $f_T$: the probability of receiving treatment (or force of treatment)
+
+with the rates of recovery:
+
+-   $r_D$: from clinical disease to asymptomatic infection
+-   $r_A$: from asymptomatic infection to sub-patent infection
+-   $r_U$: from sub-patent infection to complete recovery
+-   $r_T$: from treated clinical disease to complete recovery
+
+Superinfection may occur in individuals with asymptomatic or sub-patent infections at the same rates as standard infection (dashed arrows).
+
+The force of infection ($\Lambda_i$) is impacted by pre-erythrocytic immunity, mosquito biting rate and population size and level of infectivity (specific details can be found in the references below). All default parameters can be found in the documentation for the function `get_parameters()`. Please also note that while the infographic above displays rate parameters, the parameter list uses delays durations, e.g. the rate $r_D$ is the inverse of the delay from state *D* to *A* (`dr` in the model):
+
+$$r_D=dr^{-1}$$
+
+To maintain a constant population size during simulations, the birth rate of new susceptible individuals is set to be equal to the overall death rate.
+
+## Mosquito Biology
+
+The functions governing mosquito biological processes and dynamics are spread out between the following files:
+
+1.  [src/adult_mosquito_eqs.cpp](https://github.com/mrc-ide/malariasimulation/blob/master/src/adult_mosquito.cpp)
+2.  [src/aquatic_mosquito_eqs.cpp](https://github.com/mrc-ide/malariasimulation/blob/master/src/aquatic_mosquito.cpp)
+3.  [src/mosquito_biology.cpp](https://github.com/mrc-ide/malariasimulation/blob/master/src/mosquito_biology.cpp)
+4.  [R/mosquito_biology.R](https://github.com/mrc-ide/malariasimulation/blob/master/R/mosquito_biology.R)
+
+<img src="Model_mos.svg" style="display:block; margin: 0 auto; border-width: 0; width: 80%"/>
+
+### States
+
+Modelled mosquito states are separated into three juvenile stages: early (*E*) and late (*L*) larval stages, the pupal stage (*P*), and three adult states: susceptible (*S*<sub>M</sub>), incubating (*P*<sub>M</sub>) and infectious individuals (*I*<sub>M</sub>). Mosquitoes in any state may die, where they enter the NonExistent state. The model tracks both male and female juvenile mosquitoes, but only female adult mosquitoes.
+
+### Parameters
+
+Parameters shown on the infographic include mosquito developmental rates:
+
+-   $r_{EL}$: from early to late larval stage
+
+-   $r_{L}$: from late larval stage to pupal stage
+
+-   $r_P$: from pupal stage to adult stage
+
+Mosquito infection and incubation rates:
+
+-   $\Lambda_M$: the force of mosquito infection (FOIM)
```

Comment:
The FOIM is the force of infection on mosquitos (i.e. from human to mosquito)

Reply:
Cool - I've added that.